wpilibc/Athena/src/PWM.cpp

```diff
@@ -100,6 +101,8 @@ void PWM::SetBounds(int32_t max, int32_t deadbandMax, int32_t center,
   m_centerPwm = center;
   m_deadbandMinPwm = deadbandMin;
```

Comment:
I was debating whether keeping these around was the right thing to do, but I think it will be better to remove them, and add individual functions to the HAL to grab these, without having to allocate for the entire get bounds function. Right now, the HAL version of the grab functions are typed pointers, so I will probably just add void* versions, as otherwise every time setPWM was called the pointer would have to get cast multiple times, which just seems wasteful.